Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -3994,6 +3994,72 @@ void webkit_web_view_run_javascript_in_world(WebKitWebView* webView, const gchar
     });
 }
 
+/*
+ * webkit_web_view_run_async_javascript_function_in_world:
+ * @web_view: a #WebKitWebView
+ * @body: the JavaScript function body
+ * @arguments: a #GHashTable storing the function arguments
```

Comment:
Oops:

```suggestion
 * @arguments: a #GVariant storing the function arguments
```

Should also document the expected format of the variant, `{&sv}`. Sort of like a hash table. Then document the values: "Function argument values must be one of the following types, or contain only the following GVariant types: number, string, array, and dictionary"